core/domain/exp_services_test.py

```diff
@@ -4215,11 +4225,6 @@ def test_get_exploration_snapshots_metadata(self):
             snapshots_metadata[0]['created_on_ms'],
             snapshots_metadata[1]['created_on_ms'])
 
-        # Using the old version of the exploration should raise an error.
-        with self.assertRaisesRegex(Exception, 'version 1, which is too old'):
-            exp_services._save_exploration(  # pylint: disable=protected-access
-                second_committer_id, v1_exploration, '', [])
-
```

Comment:
This is just removed, is there any replacement?

Reply:
I tried to mimic the same thing using the `update_exploration` function.
I added a `swap_to_always_return` function so that version does not get updated in the original exploration.